runelite-client/src/main/java/net/runelite/client/plugins/mesenhanced/MesEnhanced.java

```diff
@@ -0,0 +1,246 @@
+import net.runelite.client.config.ConfigManager;
+import net.runelite.client.eventbus.EventBus;
+import net.runelite.client.plugins.Plugin;
+import net.runelite.client.plugins.PluginDescriptor;
+import net.runelite.client.plugins.PluginType;
+
+@PluginDescriptor(
+	name = "Mes Enhanced",
+	description = "1 Click MES Features -DO NOT ABUSE/AUTOCLICK THESE-",
+	type = PluginType.UTILITY
+)
+public class MesEnhanced extends Plugin
+{
+	private static final int GOBLIN_SALUTE = 2128;
+	private static final String LIGHT = "Light";
+	private static final String QUICK_BONE = "Quick Bone";
+	private static final Set<Integer> TINDER = ImmutableSet.of(
+		ItemID.TINDERBOX, ItemID.GOLDEN_TINDERBOX
+	);
+	private static final Set<Integer> LIGHTABLE_LOGS = ImmutableSet.of(
+		ItemID.LOGS, ItemID.ACHEY_TREE_LOGS, ItemID.OAK_LOGS,
+		ItemID.WILLOW_LOGS, ItemID.TEAK_LOGS, ItemID.ARCTIC_PINE_LOGS, ItemID.MAPLE_LOGS,
+		ItemID.MAHOGANY_LOGS, ItemID.YEW_LOGS, ItemID.MAGIC_LOGS, ItemID.REDWOOD_LOGS
+	);
+	private static final Set<Integer> BONES = ImmutableSet.of(
+		ItemID.BONES, ItemID.BIG_BONES, ItemID.JOGRE_BONES, ItemID.ZOGRE_BONES,
+		ItemID.BABYDRAGON_BONES, ItemID.WYRM_BONES, ItemID.DRAGON_BONES,
+		ItemID.WYVERN_BONES, ItemID.DRAKE_BONES, ItemID.LAVA_DRAGON_BONES,
+		ItemID.FAYRG_BONES, ItemID.RAURG_BONES, ItemID.HYDRA_BONES,
+		ItemID.DAGANNOTH_BONES, ItemID.OURG_BONES, ItemID.SUPERIOR_DRAGON_BONES
+	);
+
+	@Inject
+	private Client client;
+	@Inject
+	private EventBus eventBus;
+	@Inject
+	private MesEnhancedConfig config;
+	private boolean bones;
+	private boolean leftClickLog;
+	private boolean quickBone;
+	private boolean tick;
+	private boolean tinder;
+	private int bonesId;
+	private int bonesIdx;
+	private int tinderId;
+	private int tinderIdx;
+
+	@Provides
+	MesEnhancedConfig getConfig(ConfigManager configManager)
+	{
+		return configManager.getConfig(MesEnhancedConfig.class);
+	}
+
+	@Override
+	public void startUp()
+	{
+		eventBus.subscribe(AnimationChanged.class, this, this::onAnimationChanged);
+		eventBus.subscribe(ConfigChanged.class, this, this::onConfigChanged);
+		eventBus.subscribe(GameTick.class, this, this::onGameTick);
+		eventBus.subscribe(ItemContainerChanged.class, this, this::onItemContainerChanged);
+		eventBus.subscribe(MenuEntryAdded.class, this, this::onMenuEntryAdded);
+		eventBus.subscribe(MenuOptionClicked.class, this, this::onMenuOptionClicked);
+	}
+
+	@Override
+	public void shutDown()
+	{
+		eventBus.unregister(this);
+	}
+
+	private void onConfigChanged(ConfigChanged event)
+	{
+		if (!event.getGroup().equals("mesEnhanced"))
+		{
+			return;
+		}
+
+		this.quickBone = config.quickBones();
+		this.leftClickLog = config.leftClickLog();
+	}
+
+	private void onGameTick(GameTick event)
+	{
+		if (tick)
+		{
+			tick = false;
+		}
+	}
+
+	private void onAnimationChanged(AnimationChanged event)
+	{
+		if (event.getActor() != client.getLocalPlayer() || event.getActor().getAnimation() != 3705 || !bones)
```

Comment:
should 3705 be replaced with AnimationID.USING_GILDED_ALTAR ?

Reply:
Yes